### PR TITLE
API translation

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

File: ckanext/recombinant/templates/recombinant/snippets/choices_reference.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h3>{{ _("Field choices reference") }}</h3>
```
Comment (wardi) [inline ckanext/recombinant/templates/recombinant/snippets/choices_reference.html]: what's this for?

Comment (fanjinfei) [inline ckanext/recombinant/templates/recombinant/snippets/choices_reference.html]: https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/i18n/en/LC_MESSAGES/ckan.po#L6374 has html in it. Maybe we should remove it here?

Comment (wardi) [inline ckanext/recombinant/templates/recombinant/snippets/choices_reference.html]: @TkTech do you remember why the (required) block was added here? https://github.com/open-data/ckanext-canada/commit/aa297a6745ebe241fbdde7f6429b206f059d9d40#diff-3d149a3f59f2d696b95fa3a4927348b1R6372

Comment (wardi) [inline ckanext/recombinant/templates/recombinant/snippets/choices_reference.html]: If there is a reason for it we can translate the "Field choices reference" message and "(required)" separately, we already have a translation for the latter.

Comment (fanjinfei) [inline ckanext/recombinant/templates/recombinant/snippets/choices_reference.html]: ok, I will revert here.

**Pull request merged**